tsl/test/expected/cagg_permissions.out

```diff
@@ -268,3 +268,36 @@ partial_view      | _timescaledb_internal._partial_view_10
 partial_view_perm | {default_perm_user_2=arwdDxt/default_perm_user_2,default_perm_user=adDxt/default_perm_user_2}
 
 \x off
+\c :TEST_DBNAME :ROLE_SUPERUSER
+CREATE SCHEMA test;
+ERROR:  schema "test" already exists
```

Comment:
Not sure if that intentional, maybe use `IF NOT EXISTS` then

Reply:
Oops my bad... changing it to create a fresh new schema.